common/src/main/java/com/wynntils/core/features/overlays/Overlay.java

```diff
@@ -175,4 +220,22 @@ public HorizontalAlignment getRenderHorizontalAlignment() {
     public VerticalAlignment getRenderVerticalAlignment() {
         return verticalAlignmentOverride == null ? position.getVerticalAlignment() : verticalAlignmentOverride;
     }
+
+    public Vec2 getCornerPoints(Corner corner) {
+        return switch (corner) {
```

Comment:
Not anything that needs changing, just a note: why does not the compiler complain here? If we update the Corner enum, it might as well getting to a situation where it don't know what to return. That's just odd...

Reply:
This type of switch statements, (assignment and return ones) force us to implement all enums. So if there is a change in enum values, it doesn't compile, while the regular switch does.